start_site.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+cd /site
+bundle install
+bundle exec rake servequick
```

Comment:
Also, we could pass a parameter to the second command (`docker run ...`) to pick the target we'd like to run (e.g. `servequick`, `en`, `serve`). Could be an ENV variable or something like that